aten/src/THCS/generic/THCSTensor.h

```diff
@@ -15,6 +15,8 @@ typedef struct THCSTensor
     THCTensor *values;
     // Some math operations can only be performed on ordered sparse tensors
     int coalesced;
+    THCudaIntTensor *csr;
```

Comment:
Didn't realize this is an int tensor before you had to do the cwrap changes! Please make it a `LongTensor` too, that's what we use everywhere else (including `indices`).

Reply:
Okay. Should I revert cwrap changes?

Reply:
Yes, please!